README.md

```diff
@@ -954,42 +954,72 @@ Used for EV chargers.
 -|||
-in   | cmd.rated_current.get_report   | null       |                         | Get the rated or max current for ths CP
-out  | evt.rated_current.report       | int        |                         |
+in   | cmd.rated_current.get_report    | null     |                         | Get the rated or max current for ths CP
+out  | evt.rated_current.report        | int      |                         |
 -|||
-in   | cmd.current_session.get_report | null       |                         | Command for getting energy (kWh) for current session.
-out  | evt.current_session.report     | float      |                         |
+in   | cmd.current_session.get_report  | null     |                         | Command for getting energy (kWh) for current session.
+out  | evt.current_session.report      | float    |                         |
 -|||
-out  | evt.error.report               | string     |                         | 
+in   | cmd.electric_current.set        | object   |                         | Command for setting an electric current.
+in   | cmd.electric_current.get_report | null     |                         |
+out  | evt.electric_current.report     | object   |                         |
+-|||
+out  | evt.error.report                | string   |                         | 
 
 #### Interface props
 
-Name          | Value example                                                 | Description
---------------|---------------------------------------------------------------|-------------
-`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"]  | State of the CP
-`sup_access`  | ["rfid", "open", "app", "local"]                              | Access level for using CP
-`sup_errors`  | []                                                            | Error from CP
+Name          | Value example                                                                                                                            | Description
+--------------|------------------------------------------------------------------------------------------------------------------------------------------|-------------
+`sup_states`  | ["disconnected", “requesting”, “charging”, “ready\_to\_charge”, "requesting", "finished", "reserved", "unavailable", "error", "unknown"] | State of the CP
+`sup_access`  | ["rfid", "open", "app", "local"]                                                                                                         | Access level for using CP
+`sup_range`   | {"min":7.0, "max":40.0}                                                                                                                  | Optional, supported electric current range. Must be passed if the service supports current management.
+`sup_errors`  | []                                                                                                                                       | Error from CP
```

Comment:
sup_errors does not make sense from my point of view and it's probably not used. Please sync with Markus and clean this up if it's not used/required.

Reply:
Markus confirmed it's not used anywhere, I'll remove it...

Reply:
Done.